modules/installing-rhv-requirements.adoc

```diff
@@ -30,6 +30,8 @@ If you increase the number of virtual machines in the {rh-virtualization} enviro
 *** 16 GiB or more for each of the three control plane machines which provide the control plane.
 *** 16 GiB or more for each of the three compute machines, which run the application workloads.
 * The {rh-virtualization} storage domain must meet link:https://access.redhat.com/solutions/4770281[these etcd backend performance requirements].
+* 1 available physical machine per worker and 1 available host per control plane. These can both be on the same host.
```

Comment:
* At least as many physical machines as there are workers or control planes for production setups. Affinity groups can be changed for non-productive setups, but they do not guarantee redundancy.

Reply:
How many workers do you have? How many control plane nodes? (3) What's the larger number? That's how many physical nodes you need in **default configuration**. You can go lower for lab setups, but redundancy will not be guaranteed.

Reply:
@janoszen Please give feedback on changes.